keystore/README.md

```diff
@@ -0,0 +1,3 @@
+# ac_keystore
+
+This crate is a submodule of the [substrate-api-client](https://github.com/scs/substrate-api-client). It contains a fork of the sc-keystore crate and this is in order to access private functions and members of the crate.
```

Comment:
I would write the second sentence like this:

It is a fork of substrate's sc-keystore crate. The substrate-api-client needs its own fork in order to access private functions and members of the crate.

Reply:
I split the PR into two and added your suggestion to the new PR. See #757